app/uploaders/image_uploader.rb

```diff
@@ -1,6 +1,9 @@
 class ImageUploader < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick
-  storage :fog
+
+  # Choose what kind of storage to use for this uploader:
+  # storage :file
+  storage :file
```

Comment:
@tkbeili this is a different ticket that @ninalouw already did, she's just waiting os some reviews or fixing the latest things to merge it.